City-of-Helsinki/helsinki-paatokset
# PP-265 color tags change

Ticket: https://helsinkisolutionoffice.atlassian.net/browse/PP-265

Moved policymaker color css to one file, changed policymaker colors to be correct on all pages, policymaker tags on their pages are now unified.

**How to test**

Checkout branch and run `drush cim -y; drush cr`. 

Check some different type decision makers https://helsinki-paatokset.docker.so/fi/admin/content/decisionmakers. 

- Jaosto  and Toimi-/Neuvottelukunta should have as a tag LAUTAKUNNAT JA JAOSTOT and the color should be blue.
- Kaupunginhallitus tag should be HALLITUS and color should be green.
- Kaupunginvaltuusto tag should be VALTUUSTO and color should be gray.
- Viranhaltija tag should be VIRANHALTIJAT and color should be pink,
- Luottamushenkilö tag should be LUOTTAMUSHENKILÖPÄÄTTÄJÄT and color should be yellow

The colors should also be updated in https://helsinki-paatokset.docker.so/fi/paattajat
The rows in kaupunginvaltuuston jäsenet shouldn't have any color.

Check case pages, the colors should be updates there as well but the tag content should be the same as before.
